cdp_scrapers/scraper_utils.py

```diff
@@ -145,16 +145,36 @@ def parse_static_person(
                 f"{role_json['title']} is not a RoleTitle constant."
             )
         else:
-            role: Role = Role.from_dict(
-                {k: v for k, v in role_json.items() if k != "body"}
+            kwargs = {k: v for k, v in role_json.items() if k != "body"}
+            try:
+                log.debug(f"{kwargs} -> {Role.from_dict(kwargs)}")
+            except Exception:
+                pass
+            else:
+                raise NotImplementedError("We can resume using from_dict")
+
+            dt_val = kwargs.get("start_datetime")
+            kwargs["start_datetime"] = (
+                dt_val
+                if dt_val is None
+                else timezone.localize(datetime.fromtimestamp(dt_val))
             )
+            dt_val = kwargs.get("end_datetime")
+            kwargs["end_datetime"] = (
+                dt_val
+                if dt_val is None
+                else timezone.localize(datetime.fromtimestamp(dt_val))
+            )
+
+            role: Role = Role(**kwargs)
             if isinstance(role_json["body"], str):
                 role.body = primary_bodies[role_json["body"]]
             else:
                 # This role.body is a dictionary and defines a non-primary one
                 # e.g. like a committee such as Transportation
                 # that is not the main/full council
-                role.body = Body.from_dict(role_json["body"])
+                kwargs = role_json["body"]
+                role.body = Body(**kwargs)
```

Comment:
Something happened to I think `dataclasses` such that the current typing syntax used for the ingestion models, e.g. `var: str | None = None` [triggers `TypeError`](https://github.com/CouncilDataProject/cdp-scrapers/actions/runs/4335969753/jobs/7570940026#step:5:50).

This is a temporary patch that should be removed when we can just call `from_dict` again. Other instances that call `parse_static_file` will break if they update `cdp_scrapers`, but that is intentional, for the additional time zone parameter into that file.

Reply:
e.g.

```Python

from dataclasses import dataclass
from dataclasses_json import DataClassJsonMixin
from typing import Optional

@dataclass
class A(DataClassJsonMixin):
    attr_a: str = None

@dataclass
class A(DataClassJsonMixin):
    attr_a: str | None = None

>>>

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[2], line 10
      5 @dataclass
      6 class A(DataClassJsonMixin):
      7     attr_a: str = None
      9 @dataclass
---> 10 class A(DataClassJsonMixin):
     11     attr_a: str | None = None

Cell In[2], line 11, in A()
      9 @dataclass
     10 class A(DataClassJsonMixin):
---> 11     attr_a: str | None = None

TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

Reply:
I KNOW WHAT THIS IS AND IT IS ANNOYING.

dataclass_json simply doesn't allow the python 3.11 `|` -> `Union` automatic conversion.

See this issue: https://github.com/lidatong/dataclasses-json/issues/349

I think we should maybe move to a different library than dataclasses json because of this bug (and my discovery that it is slow to decode and encode) -- that is future work. To fix this current issue you can simply change the `str | None` to `Union[str, None]`